s3/pkg/conf/tidb.sql

```diff
@@ -151,7 +151,7 @@ CREATE TABLE `objectparts` (
 CREATE TABLE IF NOT EXISTS `objects` (
   `bucketname` varchar(255) DEFAULT NULL,
   `name` varchar(255) DEFAULT NULL,
-  `version` bigint(20) UNSIGNED DEFAULT NULL,
```

Comment:
Did you check if this change has any side effect on existing features

Reply:
Yes, we tested it